src/wallet.js

```diff
@@ -1192,8 +1202,11 @@ const wallet = {
    * @memberof Wallet
    * @inner
    */
-  cleanLoadedData() {
-    storage.removeItem('wallet:accessData');
+  cleanLoadedData({ cleanAccessData = true } = {}) {
+    // TODO Test with and without sending
```

Comment:
TODO?!

Reply:
Created a unit test for this and removed the TODO

Reply:
Great. Thanks!